README.md

```diff
@@ -107,90 +104,22 @@ extension RoomViewController: RoomDelegate {
 }
 ```
 
-### Screen Sharing
-
-See [iOS Screen Sharing instructions](https://github.com/livekit/client-sdk-swift/wiki/iOS-Screen-Sharing).
-
-# Thread safety
+### Something More Advanced
 
-Since `VideoView` is a UI component, all operations (read/write properties etc) must be performed from the `main` thread.
+## Documentation
 
-Other core classes can be accessed from any thread.
+LiveKit docs, guides and sample apps are available at [https://docs.livekit.io](https://docs.livekit.io). We also have minimal examples to get you started with Swift, UIKit, or Objective-C here 👉 [Starter Kits](https://github.com/livekit/client-example-collection-swift).
 
-Delegates will be called on the SDK's internal thread.
-Make sure any access to the UI is within the main thread, for example by using `DispatchQueue.main.async`.
-
-# Memory management
```

Comment:
I think these are quite useful, and I would place it further down in the details section.

Along with this, it would be good to have a section on how to keep the session running in the background (i.e. project settings, publish audio)

Reply:
Yeah, they're useful, I was thinking they could be on the wiki for the repo and perhaps linked to from the main README

Reply:
sure, that works. Just don't want to lose this. Memory management can be tricky for folks that are building real applications. If not careful, would lead to crashes/etc.